pronto_mia/lib/core/models/deployment_plan.dart

```diff
@@ -1,31 +1,24 @@
 class DeploymentPlan {
-  int get id => _id;
-  final int _id;
-
-  String get description => _description;
-  final String _description;
-
-  DateTime get availableFrom => _availableFrom;
-  final DateTime _availableFrom;
-
-  DateTime get availableUntil => _availableUntil;
-  final DateTime _availableUntil;
-
-  String get link => _link;
-  final String _link;
+  final int id;
+  final String description;
+  final DateTime availableFrom;
+  final DateTime availableUntil;
+  final String link;
+  final bool published;
 
   DeploymentPlan(
-    this._id,
-    this._description,
-    this._availableFrom,
-    this._availableUntil,
-    this._link,
-  );
+      {this.id,

```

Comment:
I think we should also refactor the other models to use named parameters, so all of them work the same way.

Reply:
Done